default/stringtables/en.txt

```diff
@@ -7037,10 +7037,55 @@ RESEARCH_TECH_GUIDE_SHORT_DESC
+[[RESEARCH_TECH_GUIDE_PARTS_OF_THE_RESEARCH_SCREEN]]
+
+[[RESEARCH_TECH_GUIDE_ALLOCATING_RESEARCH_POINTS]]
+
+[[RESEARCH_TECH_GUIDE_UTILIZING_TECHNOLOGIES]]
+
+[[RESEARCH_TECH_GUIDE_SHIP_IMPROVEMENTS]]
+'''
+
+RESEARCH_TECH_GUIDE_RESEARCH
+'''<u>Research</u>
+
+Your empire generates [[metertype METER_RESEARCH]] Points (shown as 'RP', or with the blue beaker icon) that are generated by your colonies each turn. The amount of RP that a colony produces varies depending on factors including [[encyclopedia PLANETARY_FOCUS_TITLE]], [[metertype METER_POPULATION]], [[encyclopedia ENC_SPECIAL]]s, [[encyclopedia ENC_TECH]] and [[encyclopedia ENC_SPECIES]]. RP is accumulated from all colonies in your empire without any limitations due to distance, [[encyclopedia SYSTEM_BLOCKADE_TITLE]]s, or [[metertype METER_SUPPLY]] line connections. On the main status bar across the top of your screen there is a blue beaker icon with a number that shows your total RP for your empire. If there is a warning icon next to it, it indicates that there are unallocated RP that you are not controlling how they are used. Clicking on the beaker icon will take you to the [[metertype METER_RESEARCH]] Screen.'''
+
+RESEARCH_TECH_GUIDE_PARTS_OF_THE_RESEARCH_SCREEN
+'''<u>Parts of the Research Screen</u>
+
+The [[metertype METER_RESEARCH]] Screen allows you to specify your [[metertype METER_RESEARCH]] priorities and displays what [[encyclopedia ENC_TECH]] you do and do not know.
+
+The [[metertype METER_RESEARCH]] Queue shows the progress of your current [[metertype METER_RESEARCH]] towards the technology goals that you have selected.
+
+The [[metertype METER_RESEARCH]] Display toolbar allows you to toggle various options; for example if you can see all possible [[metertype METER_RESEARCH]] targets or only those that are researchable next.
+
+The Technology Tree has individual blocks for each of the [[encyclopedia ENC_TECH]] and each block has color codes to indicate their status. Generally their overall color indicates a primary technology type; green for techs that allow better colony growth, dark blue for techs for improving colony defense, red for production techs, etc. Blocks for [[encyclopedia ENC_TECH]] that your empire knows are in brighter shades and techs that you have not finished researching are in darker shades. Lines between the technology blocks indicate prerequisite techs that lead to later techs.'''
+
+RESEARCH_TECH_GUIDE_ALLOCATING_RESEARCH_POINTS
+'''<u>Allocating Research Points (RP)</u>
+
+If you double click on an un-researched tech block that tech will be added to the queue. If prerequisite techs are also needed they will be added to the queue as well. You can have multiple different [[metertype METER_RESEARCH]] targets in the queue at once. You can drag techs up and down in the queue to change priorities, remove them from the queue or choose to add techs to the top or bottom of the queue by alt clicking on them and choosing the appropriate option.
```

Comment:
```suggestion
If you double click on an un-researched tech block that tech will be added to the queue. If prerequisite techs are also needed, they will be added to the queue as well. You can have multiple different [[metertype METER_RESEARCH]] targets in the queue at once. You can drag techs up and down in the queue to change priorities, remove them from the queue or choose to add techs to the top or bottom of the queue by alt clicking on them and choosing the appropriate option.
```

Reply:
meaning of priority and ordering unclear. indicate that techs at the top get allocations first, up to their per turn limit, with any remaining allocated to researchable techs lower in the queue